machinelearning/tensorflow.md

```diff
@@ -30,14 +30,14 @@ docker run -p 8501:8501 --name tfserving_resnet --mount type=bind,source=/tmp/re
 pip install tensorflow
 ```
 
-TensorFlow wheel supports the default eigen backend for Graviton CPUs, but typically onednn+acl provides better performance and this can be enabled by setting the below TF environment variable
+Starting from version 2.14.0, TensorFlow wheel enabled onednn+acl backend by default on Graviton CPUs which typically provides better performance than eigen backend. For previous versions of TensorFlow, onednn+acl can be enabled by setting the below TF environment variable
 ```
 export TF_ENABLE_ONEDNN_OPTS=1
 ```
 
 **Using Docker hub container**
 
-As of June 2023, Docker hub images from armswdev are based on TensorFlow 2.12.0, but also include additional downstream optimizations and experimental features. These are avaiable for trying out the experimental downstream features and provide early feedback.
+As of May 2024, Docker hub images from armswdev are based on TensorFlow 2.15.1, but also include additional downstream optimizations and experimental features. These are avaiable for trying out the experimental downstream features and provide early feedback.
```

Comment:
redundant, can delete:
"These are avaiable for trying out the experimental downstream features and provide early feedback."